docs/source/compatibility-matrix.rst

```diff
@@ -12,6 +12,12 @@ The following table shows which version of Scylla Manager restore task supports
    * - ScyllaDB Manager Version
      - ScyllaDB Open Source Version
      - ScyllaDB Enterprise Version
+   * - 3.5
+     - 5.4, 6.0, 6.1, 6.2
```

Comment:
No need to support old version of ScyllaDB here. 6.2 is good enough.
We want to encourage user to upgrade to latest - 6.2 and 2025.1 right after